pkg/bmc/redfish.go

```diff
@@ -71,5 +83,9 @@ func (a *redfishAccessDetails) DriverInfo(bmcCreds Credentials) map[string]inter
 
 // That can be either pxe or redfish-virtual-media
 func (a *redfishAccessDetails) BootInterface() string {
+	if a.isVirtualMedia {
+		return "redfish-virtual-media"
+	}
+
 	return "pxe"
```

Comment:
Ouch, why is it "pxe" here? Should be "ipxe".

Reply:
Fixed